updating/updating-disconnected-cluster.adoc

```diff
@@ -1,23 +1,131 @@
 * Have a recent xref:../backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd backup] in case your upgrade fails and you must xref:../backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restore your cluster to a previous state].
 
-//mirror module
+[IMPORTANT]
+====
+If you are upgrading from earlier release than 4.5.0, you must use the manual method of creating configmap.
 
-include::modules/update-service-overview.adoc[leveloffset=+1]
+This is because `oc` command enhancement is not available in previous releases.
+====
 
-include::modules/understanding-upgrade-channels.adoc[leveloffset=+1]
+[id="update-restricted-network"]
+== Updating a restricted network {product-title} cluster
 
-include::modules/update-upgrading-web.adoc[leveloffset=+1]
+A restricted network environment is the one in which your cluster nodes cannot access the internet. For this reason you must mirror the images to a filesystem disconnected from that environment and then bring that host or removable media across that gap.
+
+If multiple clusters present within the restricted network, the recomended way is to mirror required the release images to a single container image registry and use it for updating all the clusters. 
+
+.Procedure
+
+. Findout if there is link:https://access.redhat.com/solutions/4583231[an update path] exists between your current version and the version you want the cluster to move.
+
+. Mirror the version images to the internal container registry.
+
+** Connect the removable media to a system connected to the internet.
+** Mirror the images and configuration manifests to a directory on the removable media.
++
+----
+$ oc adm release mirror "--to-dir=${REMOVABLE_MEDIA_PATH}/mirror" quay.io/openshift-release-dev/ocp-release:{product-version}.0-x86_64
+----
++
+
+** Take the media to the disconnected environment and upload the images to the local container registry
++
+----
+$ oc image mirror "--from-dir=${REMOVABLE_MEDIA_PATH}/mirror" file://openshift/release:{product-version}.0* REGISTRY/REPOSITORY
+----
++
+. Use `oc` to log in to a given cluster to be upgraded.
++
+[NOTE]
+====
+If you are upgrading from earlier release than 4.5.0, you must must use the manual method of creating configmap.
+
+This is because `oc` command enhancement is not available in previous releases.
+====
++
+. Apply the the mirrored release image signature ConfigMap to the connected cluster
++
+----
+$ oc apply -f mirror/config/signature-sha256-81154f5c03294534.yaml
+----
+. Perform the upgrade
++
+----
+$ oc adm upgrade --allow-explicit-upgrade --to-image REGISTRY/REPOSITORY@sha256:81154f5c03294534e1eaf0319bef7a601134f891689ccede5d705ef659aa8c92
+----
+
+[id="update-restricted network-manual_configmap_creation"]
+== Manual method to create the image signature configmap
+
+The following commands will create and apply the image signature configmap to the cluster you want to update. 
```

Comment:
Every customer who needs to do this is eventually going to copy and paste each line into a file and make a script that just does the whole thing. Should we do that once for everyone and just condense this into a script that is well-documented?

Reply:
Not sure at this point. But in future we can look in to that.